Automattic/jetpack
# RNMobile: Ensure `playsInline` attribute conforms when falling back to core Video block

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/6671

## Description:

The core Video and VideoPress v5 blocks use different capitalisation for the "plays inline" setting:

* **Video:** `playsInline`  
* **VideoPress v5:** `playsinline`

Due to various technical limitations that arise from working with the app, the native implementation of VideoPress v5 in https://github.com/Automattic/jetpack/pull/35637 assigns the `playsinline` attribute when the block is added to Atomic sites, even those without VideoPress enabled. As described in https://github.com/wordpress-mobile/gutenberg-mobile/issues/6671, this leads to the setting not working as expected for Atomic sites w/o Jetpack when changed within the app.

## Proposed changes:

* https://github.com/Automattic/jetpack/pull/35981/commits/ee6267f492432a60ae30fbfb7a981fd252a57478: Rename the attribute from `playsinline` to `playsInline` when falling back to the core Video block. This will ensure the correctly capitalised setting is always used with the core block.

### Other information:


## Jetpack product discussion

N/A

## Does this pull request change what data or activity we track or use?

No, it does not.

## Testing instructions:

1. Set up an Atomic site and disable VideoPress (this can be done via the Jetpack admin panel).  
2. In the app, navigate to the site's post editor.
3. Add a video block and upload any video to it.
4. Open the block's settings panel and toggle the "plays inline" setting on.
5. Save your changes and exit the post.
6. Navigate to the post editor containing the video on the web. 
7. Verify that "plays inline" is toggled on when viewing the settings on the web.
8. Toggle the setting back off on the web to verify the changes also work on the web.
